object-mapper.rst

```diff
@@ -0,0 +1,259 @@
+Object Mapping
+==============
+
+Symfony provides a mapper to transform a given object to another one.
+
+.. _activating_the_serializer:
+
+Installation
+------------
+
+Run this command to install the ``object-mapper`` before using it:
+
+.. code-block:: terminal
+
+    $ composer require symfony/object-mapper
+
+Using the ObjectMapper Service
+------------------------------
+
+Once installed, the object mapper service can be injected in any service where
+you need it or it can be used in a controller::
+
+    // src/Controller/DefaultController.php
+    namespace App\Controller;
+
+    use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+    use Symfony\Component\HttpFoundation\Response;
+    use Symfony\Component\ObjectMapper\ObjectMapperInterface;
+
+    class DefaultController extends AbstractController
```

Comment:
As this is a demo code, wdyt using invokable controller? Make code more straight forward to me